modules/ols-creating-the-credentials-secret-using-web-console.adoc

```diff
@@ -44,7 +44,7 @@ stringData:
 ----
 apiVersion: v1
 data:
-  apitoken: NlFy 
+  apitoken: <your_api_token>
```

Comment:
I see "apitoken: <token_string_version>" used in another example so should all `apitoken` examples use the same user-replaced example?

Reply:
done